tower-http/src/services/fs/serve_dir.rs

```diff
@@ -228,7 +278,7 @@ fn append_slash_on_path(uri: Uri) -> Uri {
 }
 
 enum Output {
-    File(File, HeaderValue, usize),
+    File(File, HeaderValue, usize, Duration),
```

Comment:
Given the number of fields of this variant I think we should move to named fields:

```suggestion
    File {
        file: File,
        mime: HeaderValue,
        buf_chunk_size: usize,
        max_age: Option<Duration>,
    },
```